frontend/src/app/components/wp-query-select/wp-query-select-dropdown.component.ts

```diff
@@ -107,7 +107,8 @@ export class WorkPackageQuerySelectDropdownComponent implements OnInit, OnDestro
   private initialized = false;
 
 
-  constructor(readonly element:ElementRef,
+  constructor(private ref: ChangeDetectorRef,
```

Comment:
Please remove the space after the colon. This will probably also fix the codeclimate issue. Is there a reason you made it `private`? If not, please use `readonly`.